checker/src/main/java/org/checkerframework/checker/signedness/SignednessAnnotatedTypeFactory.java

```diff
@@ -79,6 +79,10 @@ protected void addComputedTypeAnnotations(
         super.addComputedTypeAnnotations(tree, type, iUseFlow);
     }
 
+    /**
+     * Refines the type of an integer primitive to @SignednessGlb if it is within the signed
+     * positive range (i.e. its MSB is zero).
+     */
```

Comment:
This comment doesn't describe the arguments.
Also, it suggests that the name of the method could be improved; even `addSignednessGlbAnnotation` would be better.

Reply:
@mernst I have updated the comment. Please review it.

Reply:
You didn't address my feedback, which was that there is no `@param` in the Javadoc.